docs/processing/qaqc-criteria-preprocessed.md

```diff
@@ -76,8 +81,65 @@ The carpet plot MUST appear homogeneous:
 - [ ] Check that the carpet plot is unaffected by strong dark deflections (signal drops) associated with motion peaks (particularly in the brain edge).
 - [ ] Check that no periodic modulation is visible throughout the plot.
 - [ ] Check that the brain edge has no large black bands.
-- [ ] Check for extensive signal drops that may reveal a coil failure.
+
+If any of the latter patterns is observed, flag the session and double check that the carpetplot appears homogeneous after denoising. 
 
 ### Correlations between nuisance regressors
 
 - [ ] Note down the regressors that are highly correlated and take care of not including more than one per correlated group in the nuisance regression model.
+
+# Exclusion criteria for fMRIPrep anatomical processing
+
+## Anatomical
+
+### Brain mask and brain tissue segmentation of the T1w
+
+- [ ] Check for residual intensity non-uniformity (INU).
+  If the INU is clearly visible, tweak the INU correction and re-run *fMRIPrep*.
+- [ ] Check that the brain mask does not cut off part of the brain and/or contain holes surrounding signal drop-out regions.
+  If it is not the case, exclude the data.
+- [ ] Check that the brain mask does not include parts that are clearly NOT brain.
+  Typically, this appears as bumps surrounding high-intensity areas of signal outside the brain (e.g dura or skull).
+  If it is not the case, exclude the data.
+
+### Surface reconstruction
+
+!!! note "If you used the `--fs-no-reconall` flag to skip surface-based preprocessing, the section of the report will not exist"
+
+- [ ] Check that the WM-GM boundary outlines (blue line) matches the boundary observed in the underlying image.
+- [ ] Verify that the WM and pial surface do not cross or overlap each other.
+- [ ] Verify that the pial surface (red line) does not extend past the actual pial boundary.
+
+!!! Tip "Evaluating the quality of brain surfaces"
+
+    As we will proceed with voxel-wise analysis, re-running *fMRIPrep* is necessary only when the reconstructed surfaces are extremely inaccurate.
+    That typically only happens in the presence of extreme artifacts that we should have captured previously in the step of [QA/QC for unprocessed data using *MRIQC*](../data-management/qaqc-criteria.md).
+
+# Exclusion criteria for preprocessed task fMRI
+
+!!! important "We do not put together exclusion criteria for preprocessed task fMRI"
+
+    Our QC protocol is very lenient for task fMRI as those scans have been primarily acquired for QC purposes or to aid in methodological development, rather than holding scientific significance. 
+    Task fMRI are excluded only in case of extreme distortion of the image, however such cases would have been already captured by [the first QC checkpoint](../data-management/mriqc.md#visualizing-mriqcs-individual-reports).
```

Comment:
I need more time to think about this.

Reply:
After reflection, they are almost all quality criteria (typically the presence of a coil failure or the fact that the TR is different from other sessions) apply to QCT and BHT as well IMHO.
I'm rewriting it in the next commit